gibbon.sql

```diff
@@ -2447,6 +2447,7 @@ CREATE TABLE `gibbonLibraryItem` (
   `vendor` varchar(100) NOT NULL,
   `purchaseDate` date DEFAULT NULL,
   `invoiceNumber` varchar(50) NOT NULL,
+  `cost` decimal(10,2) DEFAULT NULL,
```

Comment:
Although it seems logical, we don't update the gibbon.sql mid-version, at the end of the version we will run all the changedb entries on it before release. So, in this case, it'd be best to remove this line so it doesn't cause a clash down the road.